modules/dmrpp_module/build_dmrpp_util.cc

```diff
@@ -1080,12 +1119,29 @@ void get_chunks_for_all_variables(hid_t file, D4Group *group) {
             // that variable was synthesized (likely for CF compliance)
             H5Eset_auto2(H5E_DEFAULT, nullptr, nullptr);
             string FQN = (*btp)->FQN();
+            string name = (*btp)->name();
```

Comment:
Is this (name) and FQN above used inside this if? I can't find it, but in this code-remove mode, it's hard to tell. I see FQN used below, but it would be better to move the symbol down there or just skip it (since it only is used once - I think).

Reply:
Good catch. This is for debugging. Thanks,

Reply:
Also add a test for this feature.